api/handler/default.go

```diff
@@ -561,14 +562,34 @@ func (m ArcDefaultHandler) getTransaction(ctx context.Context, inputTxID string)
 	return nil, transactionHandler.ErrParentTransactionNotFound
 }
 
-func getPolicyFromNode() (*bitcoin.Settings, error) {
-	bitcoinRpc, err, rpcFound := gocore.Config().GetURL("peer_rpc")
-	if err != nil || !rpcFound {
-		return nil, fmt.Errorf("error getting peer_rpc from config: %v", err)
+func (m ArcDefaultHandler) getPolicyFromNode() (*bitcoin.Settings, error) {
+	peerRpcPassword := viper.GetString("peerRpc.password")
+	if peerRpcPassword == "" {
+		return nil, errors.Errorf("setting peerRpc.password not found")
+	}
+
+	peerRpcUser := viper.GetString("peerRpc.user")
+	if peerRpcUser == "" {
+		return nil, errors.Errorf("setting peerRpc.user not found")
+	}
+
+	peerRpcHost := viper.GetString("peerRpc.host")
+	if peerRpcHost == "" {
+		return nil, errors.Errorf("setting peerRpc.host not found")
+	}
+
+	peerRpcPort := viper.GetInt("peerRpc.port")
+	if peerRpcPort == 0 {
+		return nil, errors.Errorf("setting peerRpc.port not found")
+	}
```

Comment:
If this RPC configuration is loaded only when the app starts, then, in my opinion, is cleaner to pass this configuration in `NewDefaul()`. As there are multiple parameters, it would be cleaner to pass them inside a struct.
Benefits:
- Config parameters in one place.
- Visible what parameters are necessary to instantiate `api.HandlerInterface`.
- Loading/validating config parameters is not directly related to the logic of getting a policy from a node. Moving this logic out of `getPolicyFromNode()` makes it easier to test.

If `getPolicyFromNode()` could be called at run time and config parameters could be changed without restarting the app, then it would be cleaner to move the loading process of these parameters inside a method of config struct inside `handler` package.